--- origsrc/kbudget-0.6/admin/cvs.sh	2004-01-01 18:34:16.000000000 -0600
+++ src/kbudget-0.6/admin/cvs.sh	2008-06-28 23:45:53.343750000 -0500
@@ -29,7 +29,7 @@
 {
 AUTOCONF_VERSION=`$AUTOCONF --version | head -n 1`
 case $AUTOCONF_VERSION in
-  Autoconf*2.5* | autoconf*2.5* ) : ;;
+  Autoconf*2.5* | autoconf*2.5* | autoconf*2.6* ) : ;;
   "" )
     echo "*** AUTOCONF NOT FOUND!."
     echo "*** KDE requires autoconf >= 2.52"
@@ -44,7 +44,7 @@
  
 AUTOHEADER_VERSION=`$AUTOHEADER --version | head -n 1`
 case $AUTOHEADER_VERSION in
-  Autoconf*2.5* | autoheader*2.5* ) : ;;
+  Autoconf*2.5* | autoheader*2.5* | autoheader*2.6* ) : ;;
   "" )
     echo "*** AUTOHEADER NOT FOUND!."
     echo "*** KDE requires autoheader >= 2.52 (part of autoconf)"
--- origsrc/kbudget-0.6/kbudget/Makefile.am	2004-01-01 18:34:16.000000000 -0600
+++ src/kbudget-0.6/kbudget/Makefile.am	2008-06-28 23:52:54.890625000 -0500
@@ -58,5 +58,5 @@
 	 $(XGETTEXT) $$LIST -o $(podir)/kbudget.pot; \
 	fi
 
-INCLUDES = $(all_includes)
+INCLUDES = -I$(top_srcdir) $(all_includes)
 SUBDIRS =  pics
